### DISCUSSION TITLE
set current page or wake up screen

### DISCUSSION THREAD

**Discussion by TanPau1412** [Blackymas/NSPanel_HA_Blueprint#419] [Q&A] [answered]
Hi,

Is there a way to change the current display page via HA or send a command to wake up the screen.

I have an automation that puts the panel to sleep at night (as it's in my bedroom) but in the morning my routine that disables sleep mode does not light up the panel again. I have to touch the screen to bring it to life.

**Comment by lloydbayley** [comment, marked as answer]
You should be able to get home assistant to 'wake' the screen by the following command:
```
service: esphome.nspanel_dev_send_command_printf
data:
  cmd: page home
```

In sleep-mode, the screen switches to the screensaver page.
I'm still learning about this brilliant thing too but I hope I can help others along the way!

**Reply by TanPau1412** [reply]
Brilliant - this is exactly what I was hoping to achieve. Thank you !

**Reply by lloydbayley** [reply]
I'm always pleased to be able to help others in projects I've just discovered. It means I'm learning a thing or two! :)

**Comment by edwardtfn** [comment]
I'm closing this discussion as we have released v4.0 (#1057).
Please add a feature request for the missing things.

And thanks a lot for your support.
